config/config-server/src/main/resources/cruise-config.xsd

```diff
@@ -727,7 +727,7 @@
 
   <xsd:complexType name="artifactType">
     <xsd:sequence>
-      <xsd:element name="property" minOccurs="0" maxOccurs="unbounded" type="pluginPropertyType"/>
+      <xsd:element name="configuration" minOccurs="0" maxOccurs="unbounded" type="configurationType"/>
```

Comment:
I think the `maxOccurs` should be `1`, since we can have only one `<configuration>`